<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
    <item>
     <layout class="QHBoxLayout" name="videoLayout" stretch="1,3">
      <item>
       <layout class="QVBoxLayout" name="smallVideosLayout"/>
      </item>
      <item>
       <layout class="QGridLayout" name="mainVideoLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logWindow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDarkmatter_SFU_Client">
    <property name="title">
     <string>SFU</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRoom">
    <property name="title">
     <string>Room</string>
    </property>
    <addaction name="actionCreateRoom"/>
    <addaction name="actionCreateAuditRoom"/>
    <addaction name="actionDestroyRoom"/>
   </widget>
   <widget class="QMenu" name="menuParticipant">
    <property name="title">
     <string>Participant</string>
    </property>
    <addaction name="actionJoinRoom"/>
    <addaction name="actionSeek"/>
    <addaction name="actionLimit"/>
    <addaction name="actionLeaveRoom"/>
   </widget>
   <widget class="QMenu" name="menuStreams">
    <property name="title">
     <string>Streams</string>
    </property>
    <addaction name="actionStreamCamera"/>
    <addaction name="actionStreamDesktop"/>
   </widget>
   <widget class="QMenu" name="menuLastN">
    <property name="title">
     <string>LastN</string>
    </property>
    <addaction name="actionLastNone"/>
    <addaction name="actionLastOne"/>
    <addaction name="actionLastTwo"/>
    <addaction name="actionLastFour"/>
    <addaction name="actionLastAll"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionClearLog"/>
    <addaction name="actionSaveLog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDarkmatter_SFU_Client"/>
   <addaction name="menuRoom"/>
   <addaction name="menuParticipant"/>
   <addaction name="menuStreams"/>
   <addaction name="menuLastN"/>
   <addaction name="menuLog"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCreateRoom">
   <property name="text">
    <string> Create Room</string>
   </property>
  </action>
  <action name="actionCreateAuditRoom">
   <property name="text">
    <string>Create Audit Room</string>
   </property>
  </action>
  <action name="actionDestroyRoom">
   <property name="text">
    <string> Destroy Room</string>
   </property>
  </action>
  <action name="actionJoinRoom">
   <property name="text">
    <string>Join Room</string>
   </property>
   <property name="toolTip">
    <string> Join room</string>
   </property>
  </action>
  <action name="actionSeek">
   <property name="text">
    <string>Seek</string>
   </property>
  </action>
  <action name="actionLimit">
   <property name="text">
    <string>Limit</string>
   </property>
  </action>
  <action name="actionLeaveRoom">
   <property name="text">
    <string>Leave Room</string>
   </property>
   <property name="toolTip">
    <string> Leave room</string>
   </property>
  </action>
  <action name="actionStreamCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionStreamDesktop">
   <property name="text">
    <string>Desktop</string>
   </property>
  </action>
  <action name="actionLastNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionLastOne">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionLastTwo">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionLastFour">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionLastAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionSaveLog">
   <property name="text">
    <string>Save Log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
